### PR TITLE
Add Migrant Issue Shelter Map

### PR DESCRIPTION
**Pull request by asteinhart**
## What does this PR do?
adds a map using mapbox for the migrant issue stories. followed an example [here](https://docs.mapbox.com/mapbox-gl-js/example/popup-on-click/). putting up an early draft to noodle over with others.
 
pulled data from map embded on [New Arrivals Data Dashboard](https://40thward.org/cirr/new-arrivals/new-arrivals-data-dashboard/). @vernonli1 took KML file (in folder as `shelters.kml`) to convert to geopandas frame and I did some additional light cleaning and converted to geojson for mapbox all in `shelters.ipynb` file. will probably need to move these somewhere else for better organization. 

Screenshot 2024-01-17 at 2 07 01 PM

## How was the functionality tested and verified?
All steps should be completed in the order presented, and you should not move on to the next step until the previous is completed.
- [x] The title of the `html` page is the same as the chart.
- [x] The `meta_data.json` file is filled in correctly (the description is a complete sentence, the title is the same as the visual).
- [x] The chart and its associated pages have been minified using the `npm run process-visual` command.
- [x] The visual is styled similar to other charts (centered title/subtitle, no legends unless absolutely necessary, etc.)
- [x] This visual was approved by the lead developer.
- [x] The visual was approved by the story's stakeholders.

### CODE REVIEW

**Comment by michplunkett** [comment]
I'll need you to update the branch before I can look at it.

File: 2024/migrant-issue-map/meta_data.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "description": "A map of all current shelters according the city of Chicago",
```
Comment (michplunkett) [inline 2024/migrant-issue-map/meta_data.json]: Turn this into a standard sentence structure.

File: 2024/migrant-issue-map/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        <meta charset="utf-8" />
+        <meta
+            name="viewport"
+            content="initial-scale=1,maximum-scale=1,user-scalable=no"
+        />
+        <link
+            href="https://api.mapbox.com/mapbox-gl-js/v3.0.1/mapbox-gl.css"
+            rel="stylesheet"
+        />
+        <script src="https://api.mapbox.com/mapbox-gl-js/v3.0.1/mapbox-gl.js"></script>
```
Comment (michplunkett) [inline 2024/migrant-issue-map/index.html]: Add `type="text/javascript"` to this.

File: 2024/migrant-issue-map/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            }
+        </style>
+    </head>
+
+    <body>
+        <style>
+            .mapboxgl-popup {
+                max-width: 400px;
+                font: 12px/20px 'Helvetica Neue', Arial, Helvetica, sans-serif;
+            }
```
Comment (michplunkett) [inline 2024/migrant-issue-map/index.html]: Move this CSS to the inline spot a couple lines up.

**Review by michplunkett — CHANGES_REQUESTED** [review]
Great work! We discussed some issues with MapBox, they still apply here. Is there another package you can do this with? In this PR https://github.com/chicagomaroon/data-visualizations/pull/50 you have the Jupyter notebook in the `data_analysis` folder, I'd like you to add this notebook to that folder as well.

**Comment by asteinhart** [comment]
> Great work! We discussed some issues with MapBox, they still apply here. Is there another package you can do this with? In this PR #50 you have the Jupyter notebook in the `data_analysis` folder, I'd like you to add this notebook to that folder as well.

Ya going to move this over to leaflet as well!

**Review by michplunkett — APPROVED** [review]

**Review by Nik4002 — APPROVED** [review]
Looks good!

**Pull request merged**